### PR TITLE
Jest test react

### PR DESCRIPTION
**Pull request by Joanartdacosta**
- configuration of jest
- add 5 unit tests to the following components:
EmailJs
NavBar
CardService
Index
ProjectMenu

### CODE REVIEW

File: app/__tests__/CardService.test.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import { render, screen, fireEvent } from "@testing-library/react";
+import CardService from "../components/services/CardService";
+
+test("should find a button", () => {
```
Comment (doublethink13) [inline app/__tests__/CardService.test.js:4]: this test does not make sense

you are asserting the test creates a button, not that a button is created in a component

File: app/__tests__/EmailJs.test.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+test("Contacts renders correctly with a form", () => {
+  const { getByText } = render(
+    <MemoryRouter>
+      <EmailJs />
+    </MemoryRouter>
+  );
+
+  const form = document.createElement("form");
+  expect(form).not.toBeNull();
```
Comment (doublethink13) [inline app/__tests__/EmailJs.test.js:14]: you are creating a form in the test, not in the component, this is not needed

Comment (Joanartdacosta) [reply app/__tests__/EmailJs.test.js]: correction done!

File: app/__tests__/CardService.test.js (hunk 1 above)
Comment (Joanartdacosta) [reply app/__tests__/CardService.test.js]: correction done.

**Pull request merged**